code/parachain/frame/composable-maths/src/dex/constant_product.rs

```diff
@@ -465,23 +465,29 @@ pub fn compute_deposit_lp(
 ///
 /// https://github.com/ComposableFi/composable/blob/main/rfcs/0008-pablo-lbp-cpp-restructure.md#42-liquidity-provider-token-lpt-math-updates
 /// Equation 6
-pub fn compute_first_deposit_lp_<T: PerThing>(
-	pool_assets: &[(u128, u128, T)],
```

Comment:
Just removing the first `u128` param here would just solve the issue without a refactor as explained elsewhere.

Reply:
See https://github.com/ComposableFi/composable/pull/2572. I would still like to get these changes merged afterwards

Reply:
Yes, we can consider better interface/structure for this file later after we finish with Pablo overall restructure.